Tribler/Main/vwxGUI/GuiUtility.py

```diff
@@ -261,6 +287,26 @@ def ShowPage(self, page, *args):
         elif page == 'my_files':
             self.frame.librarylist.Focus()
 
+    def show_startup_splash(self, subject, changetype, objectID, *args):
+        gui_image_manager = GuiImageManager.getInstance()
+        bm = gui_image_manager.getImage(u'splash.png')
+        self.startup_splash = GaugeSplash(bm, "Loading...", self.START_GAUGE_SPLASH_TICKS)
+        self.startup_splash.Show()
+
+    def destroy_startup_splash(self, subject, changetype, objectID, *args):
+        if self.startup_splash:
+            self.startup_splash.Destroy()
```

Comment:
Just checked the Tribler codebase and it seems to be uncommon to set the variable to `None` after a call to `.Destroy()`. Guess this is ok then.